apps/webapp/Dockerfile.slim

```
@@ -6,56 +1,57 @@
+COPY --from=builder --chown=nodejs:nodejs /app .
+WORKDIR /app/apps/webapp
 
-ENV NODE_ENV=production
+ARG PORT=8080
 ENV PORT=8090
-
+ENV NODE_ENV=production
 EXPOSE 8090
 
-CMD ["node", "server.js"]
+CMD node dist/main
```

Hadolint (2.12.0)

[warning] Use arguments JSON notation for CMD and ENTRYPOINT arguments

(DL3025)